bin/common/create_k8s_certs.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+K8S_VM_IP=${K8S_VM_IP:-192.168.77.77}
+certstrap --depot-path "/run/certstrap" init --common-name "CA.kube.vagrant" --passphrase "" --years 10
```

Comment:
The path doesn't exist, but certstrap does a `mkdir -p` basically. I can add an explicit `mkdir -p` for readability though.